src/v/cluster/log_eviction_stm.cc

```diff
@@ -145,6 +140,11 @@ ss::future<> log_eviction_stm::do_write_raft_snapshot(model::offset index_lb) {
           _raft->ntp());
         index_lb = max_collectible_offset;
     }
+    vlog(
+      _logger.debug,
+      "Reaping data up until offset: {} for ntp: {}",
```

Comment:
nit: not related to this PR, but now that I look at this again, why didn't we just call this "truncate" rather than "reap"?